mpas_analysis/sea_ice/variable_stream_map.py

```diff
@@ -7,10 +7,13 @@
 seaIceVariableMap['Time'] = \
     [['xtime_start', 'xtime_end'],
      'timeSeriesStatsMonthly_avg_daysSinceStartOfSim_1',
-     'time_avg_daysSinceStartOfSim']
+     'time_avg_daysSinceStartOfSim',
+     ['xtime_startMonthly', 'xtime_endMonthly']]
```

Comment:
@pwolfram, this addition should be moved to the top so it is the highest priority.  I believe the same is true for all the added variables.